etc/yum-cron-hourly.conf

```diff
@@ -20,6 +20,9 @@ download_updates = no
 # the update to be applied
 apply_updates = no
 
+# Days of week
+days_of_week="0123456"
```

Comment:
Please add spaces before and after the `=` to make it consistent with the other options.